### ISSUE TITLE
Build failure when trying to upgrade ruby to 3.2.0 from 3.1.3

### ISSUE THREAD

**Issue by ZackMattor** [googleapis/google-cloud-ruby#20133]
Obligatory ruby upgrade issue for 3.2.0 :) 

When trying to build the latest `google-cloud-translate` gem on ruby `3.2.0` we get a build error while trying to compile grpc. It seems that they have a fix in the GRPC project... just needs to be worked up the chain probably. Let me know if there's anything I can do to help.

#### Environment details

  - OS: Fedora 33
  - Ruby version: 3.2.0
  - Gem name and version:
```
  google-cloud-translate was resolved to 3.3.0, which depends on
    google-cloud-translate-v3 was resolved to 0.5.1, which depends on
      gapic-common was resolved to 0.16.0, which depends on
        googleapis-common-protos was resolved to 1.4.0, which depends on
          grpc
```

**Comment by NivedhaSenthil** [comment]
The googleapis-common-protos depends on [grpc "~> 1.27"](https://github.com/googleapis/common-protos-ruby/blob/06ef949aa782226f4e83fa2f4a901edb5824f5c9/googleapis-common-protos/googleapis-common-protos.gemspec#L39) hence it should be installing the latest version. I am able to install translate gem without any issue on ruby 3.2.0, can you please provide the exact error you are facing ?

**Comment by dazuma** [comment]
The latest grpc (version 1.52, released Feb 6) should include a build for Ruby 3.2. Are you still seeing this problem?

**Comment by dazuma** [comment]
I believe this should be fixed with the latest grpc, so I'm closing. Feel free to reopen if it's still an issue.
